pkg/mimirtool/config/cortex.go

```diff
@@ -827,24 +827,27 @@ var removedConfigPaths = append(gemRemovedConfigPath, []string{
 	"storage.filesystem.directory", // -local.chunk-directory
 
 	// All `-swift.*` flags
-	"storage.swift.auth_url",            // -swift.auth-url
-	"storage.swift.auth_version",        // -swift.auth-version
-	"storage.swift.connect_timeout",     // -swift.connect-timeout
-	"storage.swift.container_name",      // -swift.container-name
-	"storage.swift.domain_id",           // -swift.domain-id
-	"storage.swift.domain_name",         // -swift.domain-name
-	"storage.swift.max_retries",         // -swift.max-retries
-	"storage.swift.password",            // -swift.password
-	"storage.swift.project_domain_id",   // -swift.project-domain-id
-	"storage.swift.project_domain_name", // -swift.project-domain-name
-	"storage.swift.project_id",          // -swift.project-id
-	"storage.swift.project_name",        // -swift.project-name
-	"storage.swift.region_name",         // -swift.region-name
-	"storage.swift.request_timeout",     // -swift.request-timeout
-	"storage.swift.user_domain_id",      // -swift.user-domain-id
-	"storage.swift.user_domain_name",    // -swift.user-domain-name
-	"storage.swift.user_id",             // -swift.user-id
-	"storage.swift.username",            // -swift.username
+	"storage.swift.auth_url",                              // -swift.auth-url
```

Comment:
this file contains flags that were removed in mimir 2.0 compared to cortex 1.11. Since the new flags have never been in cortex you shouldn't modify this file